### PR TITLE
[Breaking] Delete deprecated first-responder-inputs-initializer

### PR DESCRIPTION
**Pull request by SergeAstapov**
Part of #116.

Similar to #461 but focuses on one thing at a time.

### CODE REVIEW

**Comment by SergeAstapov** [comment]
Warning about `disableInputsInitializer` configuration needs to be updated

**Comment by SergeAstapov** [comment]
This is now working as expected and is ready for review.

File: index.js [hunk 1]
```diff
@@ -1,5 +1,20 @@
 'use strict';
 
 module.exports = {
-  name: require('./package').name
+  name: require('./package').name,
+
+  config(environment, appConfig) {
+    const addonConfig = appConfig['emberKeyboard'] || {};
+
+    if ('disableInputsInitializer' in addonConfig && !this.reprecationPrinted) {
```
Comment (lukemelia) [inline index.js:9]:
```suggestion
    if ('disableInputsInitializer' in addonConfig && !this.deprecationPrinted) {
```

File: index.js [hunk 2]
```diff
@@ -1,5 +1,20 @@
...
+      // Do not print deprecation message multiple times
+      // as this hook may be invoked more than once.
+      this.reprecationPrinted = true;
```
Comment (lukemelia) [inline index.js:12]:
```suggestion
      this.deprecationPrinted = true;
```

**Review by lukemelia — APPROVED** [review]

**Pull request merged**